ma/config/magento.py

```diff
@@ -11,3 +11,40 @@
 
 _LOGGER.debug("CONFIG: S=[%s] U=[%s] P=[%s] H=[%s] P=(%d)",
               SCHEME, USERNAME, PASSWORD, HOSTNAME, PORT)
+
+# Hash used to execute remote re-indexing and cache refresh.
+UTILITY_HASH = os.environ.get('MAGENTO_UTILITY_HASH', None)
```

Comment:
Caching is definitely a core component within any scalable app. However, in this case this kind of cache is not an integral part of the sync application. Moreover, cache refreshes are performed by Magento periodically so the data will eventually coalesce if it fails within our sync framework. Obviously the immediate execution is the best case scenario but not detrimental otherwise.